cpp/include/cudf/detail/stream_compaction.hpp

```diff
@@ -146,91 +146,10 @@ std::unique_ptr<experimental::table>
  */
 
 cudf::size_type unique_count(column_view const& input,
-                             bool const& ignore_nulls,
+                             include_nulls const& ignore_nulls,
```

Comment:
```suggestion
                             include_nulls const& _include_nulls,
```
...did you mean to change the parameter name here?